components/org.wso2.carbon.identity.api.user.governance/src/main/java/org/wso2/carbon/identity/user/endpoint/impl/LiteApiServiceImpl.java

```diff
@@ -90,7 +98,26 @@ public Response litePost(LiteUserRegistrationRequestDTO liteUserRegistrationRequ
                 LOG.debug("Client Error while self registering lite user ", e);
             }
             if (IdentityRecoveryConstants.ErrorMessages.ERROR_CODE_USER_ALREADY_EXISTS.getCode().equals(e.getErrorCode())) {
-                Utils.handleConflict(e.getMessage(), e.getErrorCode());
+                try {
+                    boolean isResendEnabledOnUserExistence =
```

Comment:
Shall we change the variable name to isResendCodeEnabledOnUserExistence/isResendVerificationEnabledOnUserExistence?

Reply:
Changed to `isResendVerificationEnabledOnUserExistence`